OOP/Coder/ACoder.cs

```diff
@@ -0,0 +1,138 @@
+﻿
+namespace OOP
+{
+    public class ACoder : ICoder
+    {
+
+
+        string result = "";
+        public string Encode(string text)
+        {
+
+            char[] letter = text.ToCharArray();
+
+            for (int i = 0; i < letter.Length; i++)
+            {
+                //Если не латиница и не кириллица
+                if (((int)(letter[i]) < 65) || ((int)(letter[i]) > 90) &&
+                    ((int)(letter[i]) < 97) || ((int)(letter[i]) > 122) &&
+                    ((int)(letter[i]) < 1040) || ((int)(letter[i]) > 1103))
```

Comment:
```suggestion
                if (letter[i] is (< 'A' or > 'Z') and (< 'a' or > 'z') and (< 'А' or > 'я'))
```
То же самое, но компактнее - за счёт использования нового механизма "сопоставления с образцом".